server/start.go

```diff
@@ -238,12 +240,13 @@ func startInProcess(ctx *server.Context, clientCtx client.Context, appCreator ty
 	var httpSrvDone = make(chan struct{}, 1)
 	var wsSrv rpc.WebsocketsServer
 
-	ethlog.Root().SetHandler(ethlog.StdoutHandler)
+	ethlog.Root().SetHandler(log.NewHandler(ctx.Logger))
```

Comment:
still not able to get this to work...

Reply:
i'll try
but display all evm will be used on debugging ,
in production, it will not be used

Reply:
yeah, do you think it would be better to add the `if lvl == "debug"`?

Reply:
yes, but i'll make this code work first.
we can consider later

Reply:
found the cause, fixing now